### ISSUE TITLE
Referencing VERTEX in a canvas item shader causes large offset relative to screen size.

### ISSUE THREAD

**Issue by markdibarry** [godotengine/godot#43160]
**Godot version:**
3.2.4 Beta 1 mono GLES3

**OS/device including version:**
Windows 10

**Issue description:**
Referencing `VERTEX` in a canvas item shader causes large offset relative to screen size.
![image](https://user-images.githubusercontent.com/21325943/97438777-a50eca80-18fb-11eb-87c7-6f76f56d94ad.png)
![image](https://user-images.githubusercontent.com/21325943/97438819-b3f57d00-18fb-11eb-887f-433e6a540d19.png)

**Steps to reproduce:**
1. Open a new project.
2. Add a sprite with a texture.
3. Add a shader to the canvas item with the following code:
```
shader_type canvas_item;

void vertex() {
	vec2 the_vertex = VERTEX;
}
```

**Minimal reproduction project:**
[shaderBug.zip](https://github.com/godotengine/godot/files/5451960/shaderBug.zip)

**Comment by Calinou** [comment]
@markdibarry Can you reproduce this in 3.2.3?

**Comment by markdibarry** [comment]
I cannot. This is a bug introduced in 3.2.4

**Comment by akien-mga** [comment]
Likely related to GLES3 batching.

**Comment by lawnjelly** [comment]
Likely duplicate of #42899 wind custom shader problem which was fixed by #43054.
(can't test this as away from home, but anyone with latest 3.2 build should be able to try.)

**Comment by ailernx** [comment]
I was having the same problem, but now I tried with the latest 3.2 build and it seems fixed.

**Comment by markdibarry** [comment]
I didn't know there was a new build released. I must have missed it. Sorry! Closing.

**Comment by lawnjelly** [comment]
While the fix has been merged, you would probably have to compile 3.2 dev branch yourself to make a build, until 3.2.4 beta 2 is released.

No need to be sorry, duplicate issues of new features do happen, and we would rather have them if you aren't sure it is caused by the same bug. :+1:
